functions/knative/java/with-output-binding/README.md

```diff
@@ -0,0 +1,209 @@
+## Prerequisites
+
+Knative runtime based functions can also interact with middleware via dapr components just like the Async functions.
+
+In this case, we will create two functions: `function-front` and `kafka-input`. `function-front` will send the content 
+to the kafka after a HTTP request received. The `kafka-input` will watch the kafka and consume it.
+
+You can refer to the [Installation Guide](https://openfunction.dev/docs/getting-started/installation/) to setup OpenFunction.
+
+Follow [this guide](../../../../Prerequisites.md#kafka) to install a Kafka server named `kafka-server` and a Topic named `sample-topic`.
```

Comment:
Use the prereq guide in : https://openfunction.dev/docs/getting-started/quickstarts/prerequisites/